data/remnant missions.txt

```diff
@@ -1716,3 +1725,164 @@ mission "Remnant: Broken Jump Drive job"
+	to offer
+		has "Remnant: Face to Maw 1B: done"
+	source "Aventine"
+	destination "Caelian"
+	cargo "tentacle" 1
+	on offer
+		conversation
+			`When the technician arrives in the spaceport he immediately strides over to you. "I checked the records, and my memory was correct. One of our research teams centuries ago discovered that these tentacles could be used as a devastating weapon in close quarters combat."`
+			`	He pulls out an image of a strange looking rifle and continues to chant, "The personal weapons lab under Prefect Chilia has a few of the old weapon frames. If you go visit it on Aventine, they could turn this tentacle into a potent weapon for you."`
+			choice
+				`	"That sounds good."`
+					accept
+				`	"I would rather not."`
+			`	The technician looks disappointed. "Oh well, enjoy your trophy."`
+			decline
+	on complete
+		log "People" "Chilia" `As an military prefect, Chilia is responsible for the defence of Remnant space. He can usually be found on the front lines of combat unless required to work on tactics for the Remnant as a whole.`
+		payment 250000
+		"remnant chilia" ++
+		outfit "Void Rifle"
+		conversation
+			branch chilia
+				"remnant chilia" > 1
+			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, Captain <first>," he sings. "I am Prefect Chilia. I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly. "I say 'nominal' as virtually all our activities are militaristic in some way, but usually fall under the purview of those who better understand them."`
+				goto main
+			label chilia
+			`You follow the directions provided to you and eventually find yourself at a Remnant weapons lab. You enter and are met by security, who ask you to wait. A few minutes later a fit young man emerges from another door. "Greetings, captain <first>," he sings. "We meet once again." You immediately recognize the voice as being the man who convinced you to provide the blood sample and get to know the Remnant. "We didn't get a proper introduction last time, but you have probably figured out that I have the nominal responsibility for the overall defense of Remnant space and military operations." He smiles depreciatingly, "I say 'nominal' as virtually all our activities are military in some way, but usually fall under the purview of those who better understand them."`
+			label main
+			`	He leads you to a room that looks like an armory where a team of technicians are putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
```

Comment:
```suggestion
			`	He leads you to a room that looks like an armory where a team of technicians is putting the finishing touches on a rifle stock. "The request from the technician on your behalf caught my attention, as we haven't had the key resources for making void rifles in centuries. Fortunately, we haven't really needed them since we resolved the threats that originally plagued our settlements.`
```